smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/annotations/Emitter.java

```diff
@@ -11,27 +13,28 @@
  * Emitter&lt;String&gt; emitter;
  * </pre>
  * <p>
- * You can inject emitter sending payload or {@link org.eclipse.microprofile.reactive.messaging.Message Messages}.
+ * You can inject emitter sending payload or
+ * {@link org.eclipse.microprofile.reactive.messaging.Message Messages}.
  * <p>
- * The name of the channel (given in the {@link Channel Channel annotation}) indicates which streams is fed. If must match the
- * name
- * used in a
- * method using {@link org.eclipse.microprofile.reactive.messaging.Incoming @Incoming} or an outgoing stream configured in the
- * application configuration.
+ * The name of the channel (given in the {@link Channel Channel annotation})
+ * indicates which streams is fed. It must match the name used in a method using
+ * {@link org.eclipse.microprofile.reactive.messaging.Incoming @Incoming} or an
+ * outgoing stream configured in the application configuration.
  *
- * @param <T> type of payload or {@link org.eclipse.microprofile.reactive.messaging.Message Message}.
+ * @param <T> type of payload or
+ *        {@link org.eclipse.microprofile.reactive.messaging.Message
+ *        Message}.
  */
 public interface Emitter<T> {
 
     /**
      * Sends a payload or a message to the stream.
      *
      * @param msg the <em>thing</em> to send, must not be {@code null}
-     * @return the current emitter
-     * @throws IllegalStateException if the stream does not have any pending requests, or if the stream has been
-     *         cancelled or terminated.
+     * @return the {@code CompletionStage}
```

Comment:
it needs to explain when the CS is completed.

Reply:
done